### PR TITLE
add entpoint /pages/{code}/references/{manager}

### PR DESCRIPTION
**Pull request by stefanopuddu**
- `/pages/{pageCode}`  `GET` now returns and array of `references`
-add `/pages/{code}/references/{manager}` `GET` endpoint

### CODE REVIEW

File: engine/src/main/java/org/entando/entando/web/page/PageController.java [hunk 1]
```diff
@@ -276,4 +276,11 @@ public void setAuthorizationService(PageAuthorizationService authorizationServic
         return new ResponseEntity<>(new RestResponse(page, new ArrayList<>(), metadata), HttpStatus.OK);
     }
     
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(value = "/pages/{pageCode}/references/{manager}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> getPageModelReferences(@PathVariable String pageCode, @PathVariable String manager, RestListRequest requestList) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/page/PageController.java]: Log controller methods

File: engine/src/main/java/org/entando/entando/aps/system/services/page/PageService.java [hunk 2]
```diff
@@ -638,5 +677,44 @@ public String getManagerName() {
...
+        return references;
+    }
 
+    @SuppressWarnings("rawtypes")
+    private PageServiceUtilizer<?> getPageServiceUtilizer(String managerName) {
+        Map<String, PageServiceUtilizer> beans = applicationContext.getBeansOfType(PageServiceUtilizer.class);
+        Optional<PageServiceUtilizer> defName = beans.values().stream()
+                                                     .filter(service -> service.getManagerName().equals(managerName))
+                                                     .findFirst();
+        if (defName.isPresent()) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/page/PageService.java]: Prefer assign to null and a single return

File: engine/src/main/java/org/entando/entando/aps/system/services/page/PageService.java [hunk 3]
```diff
@@ -638,5 +677,44 @@ public String getManagerName() {
         return result;
     }
 
+    private Map<String, Boolean> getReferencesInfo(IPage page) {
+        Map<String, Boolean> references = new HashMap<String, Boolean>();
+        try {
+            String[] defNames = applicationContext.getBeanNamesForType(PageUtilizer.class);
+            for (int i = 0; i < defNames.length; i++) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/page/PageService.java]: `for(String def : defNames)`

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**